frontend/next.config.ts

```diff
@@ -1,7 +1,34 @@
 import type { NextConfig } from "next";
 
+const { execSync } = require('child_process');
+
+function getGitMostRecentTag() {
+  return execOrUnknown('git describe --tags --abbrev=0 HEAD');
+}
+
+function getGitCommitHash() {
+  return execOrUnknown('git rev-parse --short HEAD');
+}
+
+function getGitCommitDate() {
+  return execOrUnknown('git show -s --format=%ci HEAD');
+}
```

Comment:
Is this commit hash and date for the latest release tag or the main branch, it feels like we may want this consistent with release tag

Reply:
Makes sense, it wasn't clear to me when this was getting calculated